src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/SomaticGenotypingEngine.java

```diff
@@ -184,6 +167,20 @@ public CalledHaplotypes callMutations(
         return new CalledHaplotypes(outputCalls, calledHaplotypes);
     }
 
+    // compute the likelihoods of: AA, AB, AC. . . where A is ref and B, C. . . are somatic alts
+    private PerAlleleCollection<Double> getHetGenotypeLogOdds(ReadLikelihoods<Allele> likelihoods,
+                                                              final boolean isNormal,
+                                                              final Strand strand) {
+        final String sample = isNormal ? matchedNormalSampleName : tumorSampleName;
+        final OptionalDouble givenAltAlleleFraction = isNormal ? OptionalDouble.of(0.5) : NO_FIXED_TUMOR_ALT_FRACTION;
```

Comment:
Change `OptionalDouble.of(0.5)` to `GERMLINE_HET_ALT_FRACTION` (currently unused) or change `NO_FIXED_TUMOR_ALT_FRACTION` to `OptionalDouble.empty().` I prefer the former.